packages/admin/README.md

```diff
@@ -61,50 +61,50 @@ Please, refer to the implementation for more details.
 
 ## Requester commands
 
-### `derive-designated-wallet`
+### `derive-sponsor-wallet`
 
 Derives the address of the [wallet designated by an Airnode for a requester](https://github.com/api3dao/api3-docs/blob/master/request-response-protocol/designated-wallet.md).
 
 ```sh
-npx @api3/airnode-admin derive-designated-wallet \
+npx @api3/airnode-admin derive-sponsor-wallet \
   --providerUrl https://ropsten.infura.io/v3/<KEY> \
-  --airnodeId 0xe1e0dd... \
-  --requester 0x9Ec6C4...
+  --mnemonic "nature about salad..." \
+  --sponsor 0x9Ec6C4...
 ```
 
-### `endorse-client`
+### `endorse-requester`
 
-[Endorses](https://github.com/api3dao/api3-docs/blob/master/request-response-protocol/endorsement.md) a client contract so that its requests can be fulfilled by the requester's designated wallet.
-The account derived from the `mnemonic` you provide here has to belong to the requester admin (master wallet).
+[Endorses](https://github.com/api3dao/api3-docs/blob/master/request-response-protocol/endorsement.md) a requester contract so that its requests can be fulfilled by the sponsor's designated wallet.
```

Comment:
These links regarding the request response protocol need to be updated as well.

Reply:
Same here. What should we link to? Should we just wait for the docs to be updated and then come back and fix these links?

Reply:
Remove the links for now maybe